tests/ocs/test_models.py

```diff
@@ -118,3 +121,31 @@ def test_value_out_of_range(self, request_group: RequestGroup):
             request.acceptability_threshold = 200
         assert exc_info.value.errors()[0]["loc"] == ("acceptability_threshold",)
         assert exc_info.value.errors()[0]["type"] == "less_than_equal"
+
+
+class TestSerialization:
+    def test_time_fields_serialized_as_mjd(self, request_group: RequestGroup):
+        """
+        Test that certain time fields on a RequestGroup are serialized as MJD.
+        This is likely specific to LCO/OCS so only the LCO Facility has this behavior.
+        """
+        facility = LcoFacility(settings=Settings(lco_token="", lco_api_root=""))
+        target = NonSiderealTarget(
+            name="mover",
+            type="ORBITAL_ELEMENTS",
+            scheme="JPL_MINOR_PLANET",
+            epochofel=datetime(2025, 1, 1),  # Time as datetime
+            orbinc=0.0,
+            longascnode=0.0,
+            argofperih=0.0,
+            eccentricity=1.0,
+            meandist=1.0,
+            meananom=0.0,
+            epochofperih=Time(2460676.5, format="jd", scale="tt"),  # Time as JD
```

Comment:
I took this example from your latest comment. While it's not an exact copy and paste, you should be able to define your target, paste in the JD into an Astropy Time object and it should still output a MJD to the LCO api.